src/Autofac/Autofac.csproj

```diff
@@ -57,6 +57,7 @@
     <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.113">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="4.7.1" />
```

Comment:
Incredibly small nit based on personal preference, can we put the 'public' package reference before the private (analyser) ones? Maybe even in a separate ItemGroup?

I'm so used to using the csproj directly these days it would be nice to group them.

Reply:
Conversely, I find it easier to locate packages when they're sorted in alpha order by package name. That's also how `dotnet add` adds the references so there's no fighting the tool. Do you feel that strongly about it?

Reply:
I don't feel strongly about it at all, like I said, personal preference.

Reply:
Sounds good, I'll leave this one, then.